Cython/Compiler/Parsing.py

```diff
@@ -3865,3 +3865,12 @@ def print_parse_tree(f, node, level, key = None):
             f.write("%s]\n" % ind)
             return
     f.write("%s%s\n" % (ind, node))
+
+def p_annotation(s):
+    """An annotation just has the "test" syntax, but also stores the string it came from
+
+    Note that the string is *allowed* to be changed/processed (although isn't here)
+    so may not exactly match the string generated by Python, and if it doesn't
+    then it is not a bug"""
```

Comment:
I like sentences to end with a dot. :)

Reply:
I'm just wondering around, but does Cython follows PEP 257?

> Unless the entire docstring fits on a line, place the closing quotes on a line by themselves.

Reply:
Cython's code base (more exactly, Pyrex's code base) is actually older than PEP-8 and PEP-257. :)
But yes, the triple quotes should be on their own next line, too.